### ISSUE TITLE
Write method to move arm to Speaker position

### ISSUE THREAD

**Issue by sinksc** [FRCTeam3206/2024-Crescendo#10]
Create or update a reusable method that can be used in either TeleOp or Autonomous to move into position to score on the speaker

Relates to:
#9 #8

**Comment by sinksc** [comment]
In the issue description or down here in the comments, you can type `#` and start typing the name of another issue to link/reference them. This could be one way to indicate priority or dependencies. For example:

Depends on #8 
Do before #8
